packages/Algernon.package/ALGDynamicWorldMenuType.class/class/extractMenuEntriesFrom..st

```diff
@@ -0,0 +1,9 @@
+algertem lists
+extractMenuEntriesFrom: aMorph
+	^ (aMorph isKindOf: DockingBarUpdatingMenuMorph) 
+		ifTrue: [ | entries |
+				aMorph updateMenu. 
+				entries := aMorph submorphs.
```

Comment:
From `Morph>>submorphs`:

> This method returns my actual submorphs collection. Modifying the collection directly could be dangerous; make a copy if you need to alter it.

I guess it should be `OrderedCollection newFrom: aMorph submorphs` to be safe